packages/bluebird-react/LICENSE.md

```diff
@@ -1,4 +1,10 @@
-# Copyright 2019 Endurance International Group, Incorporated
+# License
+
+## Multiple license notice
+
+This software includes third party open source software components React Bootstrap which have their own license. Please see [./src/react-bootstrap/LICENSE.md](./src/react-bootstrap/LICENSE.md).
```

Comment:
Not that this is a blocker for me. But just curious here are we getting this copy from legal?